packages/regl-worldview/src/Worldview.js

```diff
@@ -284,6 +269,10 @@ const Worldview = (props: Props) => (
   </ContainerDimensions>
 );
 
+Worldview.defaultProps = {
+  defaultCameraState: DEFAULT_CAMERA_STATE,
```

Comment:
Why not put this in WorldviewBase?

Reply:
Was easy to get around flow warning this way. But I guess this approach is not ideal. Will update in another PR